gitian-building.md

```diff
@@ -124,6 +124,6 @@ This will create the `.sig` files that can be committed together with the `.asse
 Gitian build.
 
 
- `./gitian-build.py --detach-sign -s $NAME $VERSION --nocommit`
+ `./gitian-build.py --detach-sign -s $NAME $VERSION --no-commit`
```

Comment:
```suggestion
 `./gitian-build.py --detach-sign --no-commit -s $NAME $VERSION`
```

to be consistent with the previous
https://github.com/bitcoin-core/docs/blob/93dc9ff1b529276e78e6f031c786b774997b39bc/gitian-building.md#L87